tests/usecases/webhook/deployment.yml

```diff
@@ -12,8 +12,8 @@ application:
           inputs:
             # TODO(dliu): Please fix "repsotiory value" and see if we can change/remove username
             # as actual developer names should not be in these files when possible.
-            username: daisy-ycguo
-            repository: https://github.com/openwhisk/wsktool.git
-            accessToken:
-            events: PUSH
-            endpoint: openwhisk.ng.bluemix.net
```

Comment:
@daisy-ycguo Please do not assume bluemix as the endpoint, but please add another comment here that uses bluemix as an example and also describes what the endpoint would look like if installed locally (native).